### PR TITLE
Sixth task.

### PR DESCRIPTION
**Pull request by tudor-chirosca**
Three arrays task.

### CODE REVIEW

File: src/main/java/firsttask/BubbleSort.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package firsttask;
 
 public class BubbleSort {
-    public static void usingBubbleSort(int[] arr) {
+    public static int[] usingBubbleSort(int[] arr) {
         for (int i = 0; i < arr.length - 1; i++) {
```
Comment (alagherasimov) [inline src/main/java/firsttask/BubbleSort.java:5]: does the task with bubble sort still work after the changes made here?

File: src/main/java/utils/ArrayUtils.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package utils;
+
+public class ArrayUtils {
+    public static int[] mergeTwoSortedArrays(int[] firstArray, int[] secondArray) {
+        int i = 0, j = 0, k = 0;
```
Comment (alagherasimov) [inline src/main/java/utils/ArrayUtils.java]: why is this method in a separate class?

File: src/main/java/sixthtask/ThreeArrays.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            int[] nums = new int[strings.length];
+            for (int j = 0; j < strings.length; j++) {
+                nums[j] = Integer.parseInt(strings[j]);
+            }
+            intList.add(nums);
+        }
+        int[] firstArr = intList.get(0);
+        int[] secondArr = intList.get(1);
+        int[] thirdArr = intList.get(2);
+        int[] initialResult = ArrayUtils.mergeTwoSortedArrays(firstArr, secondArr);
```
Comment (alagherasimov) [inline src/main/java/sixthtask/ThreeArrays.java]: there is no need in the declared above arrays
you can call them when you call the merge method
like: ArrayUtils.mergeTwoSortedArrays(list.get(X), list.get(Y);

File: src/main/java/firsttask/BubbleSort.java (hunk 1 above)
Comment (tudor-chirosca) [reply src/main/java/firsttask/BubbleSort.java:5]: Sure.
The output.txt file contains the following: "The result is: [-89, -78, -9, 0, 1, 7, 9, 17, 23, 55, 87, 333]"

File: src/main/java/utils/ArrayUtils.java (hunk 2 above)
Comment (tudor-chirosca) [reply src/main/java/utils/ArrayUtils.java]: I thought I would be able to use this method in the future.

File: src/main/java/sixthtask/ThreeArrays.java (hunk 3 above)
Comment (tudor-chirosca) [reply src/main/java/sixthtask/ThreeArrays.java]: Right.
I've made the changes.

**Pull request merged**